common/build.gradle

```diff
@@ -34,6 +34,15 @@ configurations {
 //    snapshotModImplementation {
 //        resolutionStrategy.cacheChangingModulesFor(0, "seconds")
 //    }
+    apiElements {
```

Comment:
Might want to use common versions here (instead of fabric). Not 100% sure though

Reply:
doesn't exist for the versions.